akka-docs/src/test/scala/typed/tutorial_3/DeviceInProgress.scala

```diff
@@ -38,7 +38,7 @@ object DeviceInProgress2 {
     //#read-protocol-2
     sealed trait Command
     final case class ReadTemperature(requestId: Long, replyTo: ActorRef[RespondTemperature]) extends Command
-    final case class RespondTemperature(requestId: Long, value: Option[Double])
+    final case class RespondTemperature(requestId: Long, value: Option[Double]) extends Command
```

Comment:
PR already closed, but for the record, it's a reply not a command

Reply:
@patriknw , thanks for the review. Yes, I realized after I read down the document.  😅